src/main/java/com/product/review/repository/ProductReviewRepository.java

```diff
@@ -0,0 +1,24 @@
+package com.product.review.repository;
+
+import java.util.List;
+import java.util.Optional;
+
+import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.data.jpa.repository.Query;
+import org.springframework.data.repository.query.Param;
+import org.springframework.stereotype.Repository;
+
+import com.product.review.domain.Review;
+
+
+@Repository
+public interface ProductReviewRepository extends JpaRepository<Review, Integer>{
+
```

Comment:
We can simpify this using findBy* methods

public List<Review> findReviewByProductId(int productId);